process-wpt-results.js

```diff
@@ -213,33 +217,47 @@ export function score_run (run, against_run, focus_areas_map) {
         const run_test = run.test_scores[test]
 
         // score new tests not present in older runs
-        if (!run_test) continue
+        if (!run_test) {
+          for (const area of areas) {
+              scores[area].total_subtests += 1
+          }
+          continue
+        }
 
         const subtest_names = Object.keys(subtests)
         if (!subtest_names.length) {
             for (const area of areas) {
                 scores[area].total_score += run_test.score
+                scores[area].total_subtests += 1
+                scores[area].total_subtests_passed += run_test.score
             }
         } else {
-            let test_score = 0
+            let subtests_count = 0
```

Comment:
Isn't this just the same as `subtest_names.length`? I don't think we need a counter.

Reply:
True, I can remove that one.